integrated_channels/integrated_channel/exporters/course_metadata.py

```diff
@@ -104,3 +105,14 @@ def resolve_removed_courses(self, *args, **kwargs):  # pylint: disable=unused-ar
         courses = self.courses if not self.removed_courses_resolved else {}
         self.removed_courses_resolved = True
         return courses
+
+    def format_title(self, course_run):
+        """
+        Return the transformed version of the course title, as well as the locale. For all instructor-paced courses
+        also include the start date to distinguish multiple runs of the same course (ENT-782)
+        """
+        title = course_run.get('title') or ''
+        course_run_start = parse_lms_api_datetime(course_run.get('start'))
+        if course_run_start and course_run.get('pacing_type') == 'instructor_paced':
+            title += ' (Starts: {:%B %Y})'.format(course_run_start)
+        return title
```

Comment:
One more slight tweak to get this right I think:
```
title = course_run.get('title') or ''
course_run_start = course_run.get('start')
if course_run_start and course_run.get('pacing_type') == 'instructor_paced':
    title += ' (Starts: {:%B %Y})'.format(
        parse_lms_api_datetime (course_run_start)
    )
return title
```

Reply:
Perfect, that does look better, I was worried about side-effects changing parse_lms_api_datetime for the `None` case 👍